doc/installation.md

```diff
@@ -22,7 +22,7 @@ it out, we have provided the instructions and demo. Play around with it. Any bug
    docker run -it -p 8888:8888 sqlflow/sqlflow:latest
    ```
 
-1. Open a web browser, go to `localhost:8888` and paste the token output from Notebook command above.  Select the "New" drop-down menu on the right side, and open the "Python 3" development environment in a new Notebook cell (also in a new tab). In the new cell, type in below SELECT statement to fetch 5 records from train table in Iris database. 
```

Comment:
I think we cannot remove the phrase of "paste the token printed by the Notebook server".

Reply:
Maybe we don't need the token authorization here? Users don't need to find the token text from the Docker container logs (local computer) or the Pod logs (k8s), it's easier to use. And I removed the token auth: https://github.com/sql-machine-learning/sqlflow/pull/475/files#diff-ce186285de1e4082474a97a1fa4b74ecR47